### PR TITLE
Update Ansible References

### PR DESCRIPTION
**Pull request by cerberushades**

### CODE REVIEW

File: docs/maintain/validate/run-validator-ansible.md [hunk 1]
```diff
@@ -464,7 +464,7 @@ heimdallcli generate-validatorkey ETHEREUM_PRIVATE_KEY
 ```
 
 :::note
-* ETHEREUM_PRIVATE_KEY — your Ethereum private key.
+* ETHEREUM_PRIVATE_KEY — your Ethereum wallet address.
```
Comment (DannyS03) [inline docs/maintain/validate/run-validator-ansible.md:467]: private key is correct here

File: docs/develop/network-details/full-node-deployment.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 id: full-node-deployment
-title: Set up a Full Node
+title: Full Node Ansible
```
Comment (DannyS03) [inline docs/develop/network-details/full-node-deployment.md:3]: why change this to Ansible deployment?

**Review by DannyS03 — CHANGES_REQUESTED** [review]

**Comment by DannyS03** [comment]
I don't agree that we change the Full Node Deployment to Ansible as there is already a guide on deploying with Ansible (using Ansible for "full node deployment" can be presented as a tool in the steps, if anything, and the script should update the parameter name because `ETHEREUM_PRIVATE_KEY` shouldn't = "wallet address", despite SME request @Nazeim

**Comment by sunil0410** [comment]
So the names contradict the Binaries part we have clearly mentioned Full Node Binaries but in the case of Ansible the naming convention should be similar to Full Node Ansible.

image

**Comment by sunil0410** [comment]
image

Also, that's the standard we maintain with the Validator Nodes and the Setup a Full Node keyword looks a bit odd.

**Comment by sunil0410** [comment]
For the next change regarding the wallet address again there is some contradiction

Binaries:
https://docs.polygon.technology/docs/maintain/validate/run-validator-binaries/#generating-a-heimdall-private-key

Ansible:
https://docs.polygon.technology/docs/maintain/validate/run-validator-ansible/#generate-a-heimdall-private-key

The information used must be similar for both the cases Binaries as well as Ansible steps but it's vice-versa.

It's only the wallet address that should be used.

image
image

**Comment by DannyS03** [comment]
thanks @sunil0410 - right I see what you mean with the node deployment.

For the wallet address change, isn't it contradictory having the parameter called `ETHEREUM_PRIVATE_KEY` hold the wallet address; shouldn't it be called `ETHEREUM_PUBLIC_KEY`, in that case?

**Pull request merged**